.gitignore

```diff
@@ -3,3 +3,8 @@ lerna-debug.log
 npm-debug.log
 yarn-error.log
 packages/*/yarn.lock
+
+# Editors stuff
+.idea
+.project
+.vscode
```

Comment:
Please don't add to the gitignore file. This should be in your global .gitignore file for your machine.